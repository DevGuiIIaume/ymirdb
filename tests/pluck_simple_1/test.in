set a 1 2 3 4 5
pluck a -1
pluck a 0
pluck a 6
pluck a 1
get a
pluck a 5
pluck a 4
get a
pluck a 3
get a
pluck a 1
pluck a 1
get a
pluck a 0
pluck a 2
pluck a 1
bye
