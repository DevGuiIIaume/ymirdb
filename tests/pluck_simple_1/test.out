> ok

> index out of range

> index out of range

> index out of range

> 1

> [2 3 4 5]

> index out of range

> 5

> [2 3 4]

> 4

> [2 3]

> 2

> 3

> []

> index out of range

> index out of range

> index out of range

> bye
